plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -229,7 +229,7 @@ export class EventsProcessor {
             topic: this.pluginsServer.CLICKHOUSE_JSON_EVENTS_KAFKA_TOPIC,
             key: uuid,
             value: Buffer.from(JSON.stringify(rawEvent)),
-            waitForAck: true,
+            waitForAck: false,
```

Comment:
I think this PR uncovered something unexpected? I was having issues with `promise.catch` not being a method, because the values at runtime were actually already realized values.

It seems like we were waiting for acks deep down in the stack already, and so the array of promises we were awaiting were already realized values?